docs/src/contribute/pull-requests.md

```diff
@@ -53,45 +53,9 @@ git add -A
 git commit
 ```
 
-All ESLint projects follow [Conventional Commits](https://www.conventionalcommits.org/) for our commit messages. (Note: we don’t support the optional scope in messages.)  Here's an example commit message:
+The commit messages do not need to be in any specific format because they do not show up in the changelog.
```

Comment:
I still think we should be encouraging people to format their commit messages in Conventional Commits format. I'd rather not need to manually edit pull request titles all the time.

Reply:
I see. Then it would be good to keep information about the commit message here and describe what is specific to the ESLint PR process. I'll change this.

Reply:
Reverted the part about Conventional Commits, should be okay now.